Added Rule to add issue URL

frankhu-2021 opened the pull request:
Added Rule to add issue URL in content.js
https://github.com/craigshoemaker/spineedit/issues/4

Commit:
Added Rule to add issue URL

Added Rule to add issue URL in content.js

Commit:
Update content.js

Missing semicolon

craigshoemaker commented on src/content.js:
```diff
@@ -76,6 +79,7 @@ const domains = {
       { apply: url => url.replace(/\/(.*)-docs\//, '$1-docs-pr/') },
 
       { apply: url => commonRules.addDescription(url) },
+      { apply: (url, issueURL) => commonRules.addIssueURL(url , issueURL) },
```
Since you're calculating the issue URL in the function, you don't need to pass it in as an argument.

```suggestion
      { apply: url => commonRules.addIssueURL(url) },
```

craigshoemaker commented on src/content.js:
```diff
@@ -4,7 +4,10 @@ let domain;
 const commonRules = {
   addDescription: url => `${url}?description=`,
-
+  addIssueURL: (url, issueURL) =>{
```
Since this only applies to the GitHub implementation, you can just move this down into the rules specifically for GitHub.

frankhu-2021 replied:
okay, I've gone ahead and made that change. If you approve, should i just make the one commit to the master branch? 

I feel like there's a lot of commits here that are unnecessary.

frankhu-2021 replied:
https://github.com/craigshoemaker/spineedit/pull/5/commits/4ffbc7aa4acb680119cacd951891434cec241cb9

craigshoemaker replied:
If you want to squash your commits and then merge, feel free. I am fine if you just want to merge as-is as well.

craigshoemaker reviewed (changes requested):
Just a few tweaks 😉

Commit:
Update src/content.js

Co-Authored-By: Craig Shoemaker <craig@craigshoemaker.net>

Commit:
Update content.js

fixing per git issue comment : https://github.com/craigshoemaker/spineedit/pull/5#discussion_r335545479
